AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/HostedUI/HostedUIASWebAuthenticationSession.swift

```diff
@@ -18,7 +18,7 @@ class HostedUIASWebAuthenticationSession: NSObject, HostedUISessionBehavior {
                       inPrivate: Bool,
                       presentationAnchor: AuthUIPresentationAnchor?,
```

Comment:
The entire thing would not be used since the API is wrapped around iOS and macOS, any way to just not include the entire file?

Reply:
This might be left-over from when watchOS was supported :P. I'll check